Sources/Service/Environment/Environment+DotEnv.swift

```diff
@@ -0,0 +1,65 @@
+
+import Core
+#if os(Linux)
+    import Glibc
+#else
+    import Darwin
+#endif
+
+extension Environment {
+    /// Loads the ".env" and the ".env.environment" files if they exist
+    public func loadDotEnv() {
+        // Load the default one
+        self.loadDotEnvFile(".env")
+        
+        // Load any environment specific ones next
+        self.loadDotEnvFile(".env.\(self.name)")
+    }
+    
+    /// Loads the specific env file and sets the appropiate environment variables
+    public func loadDotEnvFile(_ filename: String) {
```

Comment:
oh, it's here ... `func load(dotEnvFile filename: String)` would be more "swifty"